VotingApplication/VotingApplication.Web/Views/Routes/Manage.cshtml

```diff
@@ -71,15 +71,11 @@
                 <h4 class="icon-padded-left">Expires:</h4>
             </div>
             <div>
-                <span ng-if="account" class="icon-padded-left">
-                    <quick-datepicker ng-model='poll.ExpiryDate' on-change="poll.Expires=(poll.ExpiryDate!=null);updatePoll()" placeholder="Never" date-format="dd/MM/yyyy" date-filter="dateFilter"></quick-datepicker>
-                </span>
-                <span ng-if="!account">
+                <span>
                     <span class="icon-padded-left">{{formatPollExpiry()}}</span>
```

Comment:
The formatPollExpiry needs to cut the seconds off the time. It should also be using short-text months (Mar, Jan, Jul) rather than 3/25/2015